index.js

```diff
@@ -0,0 +1,377 @@
+const express = require('express'); // pull in express
+const cors = require('cors');
+const server = express(); // instantiate server
+const logger = require('morgan');
+const db = require('./data/helpers/userDb');
+const postDb = require('./data/helpers/postDb');
+const port = 5201;
+
+
+const capitalize = (req, res, next) => {
+  const { name } = req.body;
+  req.body.name = req.body.name.charAt(0).toUpperCase() + req.body.name.slice(1);
+  next();
+};
+
+const textRecognize = (req, res, next) => {
+  const newText = req.body.text;
+  req.text = newText
+};
+
+
+server.use(cors()); // connects react
+server.use(express.json()); 
+server.use(logger('combined'));
+
+
+
+server.listen(port, () => {
+  console.log(`================Running on port ${port}==============`);
+});
+
+
+
+
+/*                        USERS                              */
+
+server.get('/', (req, res) => {
+  res.send('<h3>Welcome to INSTANT-GRAM!</h3>');
+});
+
+server.get('/users', (req, res) => {
+  db.get()
+    .then(users => {
+      console.log(' I am users name: ', users);
+      res.json(users);
+    })
+    .catch(err => res.send(err))
+});
+
+
+
+
+
+
+
+
+server.post('/users', capitalize, (req, res) => {    
```

Comment:
We use a middleware to capitalize, how about a middleware to handle our errors instead of having these checks within our post request. It's a matter of preference, but consider that we'll be reusing these same checks elsewhere, such as on a put request. Using a custom middleware allows us to be more DRY